### PR TITLE
Use react portal

### PR DESCRIPTION
**Pull request by console-bob**

### CODE REVIEW

File: test/Drawer.test.js [hunk 1]
```diff
@@ -2,19 +2,22 @@ import React              from 'react';
...
 
 import { Drawer, BasicView, DetailView } from '../index';
+import DrawerPortal from '../src/js/DrawerPortal';
 
 const text = {
                 headerTitle       : "Basic Title",
                 closeButtonSRText : "Close",
                 backButtonText    : "Back",
-             };
+             };``
```
Comment (nlkluth) [inline test/Drawer.test.js]: is this a typo?

**Comment by nlkluth** [comment]
Not sure who owns this stuff now but you might want to tag someone from that team 😃

File: test/Drawer.test.js (hunk 1 above)
Comment (console-bob) [reply test/Drawer.test.js]: Yes. Fixed.

**Comment by console-bob** [comment]
oops. I agree. I added Greg Davis to the reviewers list. Thanks.

**Pull request merged**